test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterHttpServerTests.cs

```diff
@@ -105,5 +105,44 @@ public async Task PrometheusExporterHttpServerIntegration()
                 "^# TYPE counter_double counter\ncounter_double{key1='value1',key2='value2'} 101.17 \\d+\n$".Replace('\'', '"'),
                 await response.Content.ReadAsStringAsync().ConfigureAwait(false));
         }
+
+        [Theory]
+        [InlineData("http://example.com")]
+        [InlineData("https://example.com")]
+        public void ServerEndpointSanityCheckPositiveTest(string uri)
+        {
+            using MeterProvider meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddPrometheusExporter(opt =>
+                {
+                    opt.HttpListenerPrefixes = new string[] { uri };
+                })
+                .Build();
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        [InlineData("ftp://example.com")]
+        public void ServerEndpointSanityCheckNegativeTest(string uri)
+        {
+            try
+            {
+                using MeterProvider meterProvider = Sdk.CreateMeterProviderBuilder()
+                    .AddPrometheusExporter(opt =>
+                    {
+                        opt.HttpListenerPrefixes = new string[] { uri };
+                    })
+                    .Build();
+            }
+            catch (Exception ex)
+            {
+                Assert.Equal("System.ArgumentException", ex.GetType().ToString());
```

Comment:
Would something like this work?
```suggestion
                Assert.Equal(typeof(System.ArgumentException), ex.GetType());
```